pillar/redash.sls

```diff
@@ -9,5 +9,26 @@ apache:
 
 docker:
   docker_compose:
-    # 1.22.0 is used in https://github.com/getredash/setup/blob/cb47626b6823cbafac407b3e8991e97f53121f6e/setup.sh#L18
-    version: 1.22.0
\ No newline at end of file
+    version: 1.29.2
+
+redash:
+  docker_tag: redash/redash:10.0.0-beta.b49597
+  postgres:
+    username: redash_user
+    hostname: ocp14.open-contracting.org
```

Comment:
Why can't it connect on localhost?

Some docker-compose files use `network_mode: "host"`, but I know this has its own disadvantages.